### PR TITLE
feat/build: print configure command on failure

### PR DESCRIPTION
**Pull request by afck**
This change is [Reviewable](https://reviewable.io/reviews/maidsafe/rust_sodium/18)

### CODE REVIEW

File: rust_sodium-sys/build.rs [hunk 1]
```diff
@@ -68,15 +68,16 @@ fn main() {
 
     let command = "(New-Object System.Net.WebClient).DownloadFile(\"".to_string() + &url +
                   "\", \"" + &gz_path + "\")";
-    let download_output = Command::new("powershell")
-        .arg("-Command")
+    let mut download_cmd = Command::new("powershell");
```
Comment (Fraser999) [inline rust_sodium-sys/build.rs:71]: Just commenting to test the new review tool.  Although this diabolical use of `cmd` as opposed to `command` is worth commenting on!

File: rust_sodium-sys/build.rs [hunk 2]
```diff
@@ -117,14 +118,14 @@ fn main() {
 
     // Get path to gcc in order to guess location of libpthread.a
     let mut lib_search_dirs = vec![Path::new(&install_dir).join("lib")];
-    let where_output = Command::new("where")
-        .arg(gcc::Config::new().get_compiler().path())
+    let where_cmd = Command::new("where");
+    let where_output = where_cmd.arg(gcc::Config::new().get_compiler().path())
```
Comment (Fraser999) [inline rust_sodium-sys/build.rs:122]: Where "command" indeed!

**Review by Fraser999 — APPROVED** [review]

File: rust_sodium-sys/build.rs (hunk 1 above)
Comment (afck) [reply rust_sodium-sys/build.rs:71]: New review tool works perfectly: `cmd` had the desired effect. :-D

**Pull request merged**